src/modules/dashboard/components/ActionsPage/ActionsPage.css

```diff
@@ -5,11 +5,23 @@
   background-color: white;
 }
 
-.container {
-  display: flex;
-  align-items: flex-start;
-  justify-content: space-between;
+.recoveryTag {
+  margin-bottom: 10px;
+  padding: 4px 8px;
+  height: 20px;
+  width: 69px;
```

Comment:
I don't think we need a fixed width, as we have no strict requirement for it's width. 

The padding will take care of that anyway.

Imagine this some other way. At some point, internationalization will kick in and we'll have multiple languages. Longer works will break this.

Just remove this and it's all fine.